### PR TITLE
Announcement for Fine beta

### PR DESCRIPTION
**Pull request by JPrause**

### CODE REVIEW

**Comment by JPrause** [comment]
@mfeifer when for have a few minutes, can you review.

File: site/_data/releases.yaml [hunk 1]
```diff
@@ -3,10 +3,10 @@ stable:
...
   filename: euwe-2
 
-# prerelease:
-#  name: Euwe-1-RC2
-#  tag: euwe-1-rc2
-#  filename: euwe-1-rc2
+prerelease:
+ name: Fine-1-Beta
+ tag: fine-1-beta
+ filename: fine-1-beta
```
Comment (cybette) [inline site/_data/releases.yaml]: I notice in the builds that the tag/filename is `fine-1-beta1` instead of `fine-1-beta`

**Comment by JPrause** [comment]
Thanks @cybette, update made.

File: site/_posts/2017-04-03-Announcing-Fine-Beta-Release.html.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Take a look at the [changelog](https://github.com/ManageIQ/manageiq/blob/fine/CHANGELOG.md/) for more details.
+
+We encourage you to download the Fine Beta Release and try it out!
+
+
+| Appliance | Format | Size |
+| --------- | ------ | ---- |
+| [Docker](https://hub.docker.com/r/manageiq/manageiq/) | docker | 0.5 GB |
+| [Google Compute Engine](http://releases.manageiq.org/manageiq-gce-fine-1-beta1.gz) | gce | 1.1 GB |
```
Comment (cybette) [inline site/_posts/2017-04-03-Announcing-Fine-Beta-Release.html.md]: Sorry I missed this earlier, looks like file ext for GCE is tar.gz instead of just gz

Comment (JPrause) [inline site/_posts/2017-04-03-Announcing-Fine-Beta-Release.html.md]: Thanks again.

**Comment by simaishi** [comment]
@JPrause https://github.com/ManageIQ/manageiq.org/pull/495 added GCE and EC2 to the download page. I see GCE in the announcement page, but not EC2. We might want to add that too...

**Comment by JPrause** [comment]
Thanks @simaishi, made the change to add EC2

**Comment by JPrause** [comment]
@cybette got the all-clear from DaJo. We can merge this to announce Fine-Beta.

**Comment by JPrause** [comment]
@cybette before merging, let me change the date.

**Comment by JPrause** [comment]
@cybette okay, date changed. Clear for merge.

**Pull request merged**